### ISSUE TITLE
what is the setting of the param :DOUBLE_BIAS and WEIGHT_DECAY

### ISSUE THREAD

**Issue by Usernamezhx** [xanderchf/MonoDepth-FPN-PyTorch#13]
First of all. thanks for your code. I don't know the setting about hte param : DOUBLE_BIAS and WEIGHT_DECAY . it show me that:

**[epoch  0][iter   10] loss: nan RMSElog: nan grad_loss: nan normal_loss: nan
[epoch  0][iter   20] loss: nan RMSElog: nan grad_loss: nan normal_loss: nan
[epoch  0][iter   30] loss: nan RMSElog: nan grad_loss: nan normal_loss: nan
[epoch  0][iter   40] loss: nan RMSElog: nan grad_loss: nan normal_loss: nan
[epoch  0][iter   50] loss: nan RMSElog: nan grad_loss: nan normal_loss: nan
[epoch  0][iter   60] loss: nan RMSElog: nan grad_loss: nan normal_loss: nan
[epoch  0][iter   70] loss: nan RMSElog: nan grad_loss: nan normal_loss: nan**

**Comment by Usernamezhx** [comment]
I find the question. because I use my own data. and the data has some zero value .So when I use the torch.log function. it will produce the inf value.

**Comment by Haiyan-Chris-Wang** [comment]
what number you set about the DOUBLE_BIAS and WEIGHT_DECAY?

**Comment by vinojhosan** [comment]
what number you set about the DOUBLE_BIAS and WEIGHT_DECAY?
